From f5f9df24038a0457730d1d420d1edd93d737560e Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Fri, 22 Sep 2017 11:30:46 +0800
Subject: [PATCH 202/205] arm: dts: am33xx: enable gpmc for LCD etc.

gpmc should be enabled by default since a lot of devices depend on it.

(cherry picked from commit 0bd0c6fd0307237064c28535eedcbcd0bb33e44a)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/boot/dts/am335x-evmsk.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-evmsk.dts b/arch/arm/boot/dts/am335x-evmsk.dts
index b153440..7dd0e55 100644
--- a/arch/arm/boot/dts/am335x-evmsk.dts
+++ b/arch/arm/boot/dts/am335x-evmsk.dts
@@ -448,6 +448,10 @@
 	status = "okay";
 };
 
+&gpmc {
+	status = "okay";
+};
+
 &i2c0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c0_pins>;
-- 
1.7.9.5

